### ISSUE TITLE
@slidy/core: `end` custom event

### ISSUE THREAD

**Issue by EricRovell** [Valexr/Slidy#67]
In case the `loop` mode is not used, it makes sense to have a `on:end` custom event for the core packages.
@Valexr , what do you think about it?

**Comment by Valexr** [comment]
> In case the `loop` mode is not used, it makes sense to have a `on:end` custom event for the core packages.
> @Valexr , what do you think about it?

`on:end` What? 😁

**Comment by EricRovell** [comment]
It would indicate that the navigation finishes. Would be useful, for example, to use it for lazy loading more slides.

**Comment by Valexr** [comment]
> It would indicate that the navigation finishes. Would be useful, for example, to use it for lazy loading more slides.

When navigation finished? I’m don’t understand what you meant…🤷🏻‍♂️

**Comment by EricRovell** [comment]
> > It would indicate that the navigation finishes. Would be useful, for example, to use it for lazy loading more slides.
> 
> When navigation finished? I’m don’t understand what you meant…🤷🏻‍♂️

When you navigate to the last item.

**Comment by Valexr** [comment]
> > > It would indicate that the navigation finishes. Would be useful, for example, to use it for lazy loading more slides.
> > 
> > 
> > When navigation finished? I’m don’t understand what you meant…🤷🏻‍♂️
> 
> When you navigate to the last item.

I think this task already done🤓 info about last item available in the `on:move` event in `event.detail.index` 🤷🏻‍♂️

**Comment by EricRovell** [comment]
You are correct. It can be implemented by user, Still, even if it is simple, I think it would be quite useful to have this kind of event. Let this issue live for some time, maybe someone will find it and ask for it. If not, we will close it.
